src/index.js

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
 import './index.css';
+import { Provider } from 'react-redux';
 import reportWebVitals from './reportWebVitals';
 import App from './App';
+import store from './Redux/store';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+  <Provider store={store}>
     <BrowserRouter>
       <App />
     </BrowserRouter>
-  </React.StrictMode>,
+  </Provider>,
```

Comment:
Though i had a question, as to why you removed the React.StrictMode

Reply:
React.StrictMode is used to render pages twice, but we don't need that and it is common practice to remove the <React.StrictMode> wrapper before deploying your application to production.